File: src/main/java/team4/footwithme/chat/service/response/ChatroomResponse.java
package team4.footwithme.chat.service.response;

import team4.footwithme.chat.domain.Chatroom;

public record ChatroomResponse(
        Long chatroomId,
        String name) {

    public ChatroomResponse(Chatroom chatroom) {
        this(
                chatroom.getChatroomId(),
                chatroom.getName()
        );
    }
}

File: src/main/java/team4/footwithme/chat/service/response/ChatroomResponse.kt
package team4.footwithme.chat.service.response

import team4.footwithme.chat.domain.Chatroom

@JvmRecord
data class ChatroomResponse(
    val chatroomId: Long?,
    val name: String?
) {
    constructor(chatroom: Chatroom?) : this(
        chatroom!!.chatroomId,
        chatroom.name
    )
}
